packages/cli/README.md

```diff
@@ -67,7 +71,15 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
    };
    ```
 
-4. Define your custom categories.
+4. Run the CLI with `npx code-pushup` (see `--help` for list of commands and arguments).
+
+5. View report file(s) in output directory (specified by `persist.outputDir` configuration).
+
+<img src="./docs/images/cli-run-stdout-example.png" width="400" alt="example of code-pushup terminal output">
```

Comment:
Not sure I'd include this screenshot here, then contents are misleading (no categories or example plugins mentioned in above setup).

Reply:
removed